python/lsst/validate/base/__init__.py

```diff
@@ -5,15 +5,15 @@
 
 try:
     # version.py is auto-generated by sconsUtils
-    from .version import *  # noqa: F403
+    from .version import *  # noqa: F401, F403
```

Comment:
Good idea. I guess we could update https://developer.lsst.io/coding/python_style_guide.html#flake8-configuration-files to add guidance on ignoring certain types of files (certainly we want to do this for version.py and there are some other ignores I needed to add to setup.cfg). I suppose I should RFC this?